Breeze.Sharp/MetadataStore.cs

```diff
@@ -40,6 +40,8 @@ public class MetadataStore : IJsonSerializable {
 
     public static String MetadataVersion = "1.0.3";
 
+    private Regex regExGeneric = new Regex(@"`\d+");
```

Comment:
Entitities from Asp.Net Identity contain generics and in metadata I have in class name `1 for example.

Do you know how to resolve in another way?